src/schema/v2/conversation/collectorResume.ts

```diff
@@ -0,0 +1,18 @@
+import { GraphQLObjectType, GraphQLBoolean } from "graphql"
+import { CollectorProfileType } from "schema/v2/CollectorProfile/collectorProfile"
+import { ResolverContext } from "types/graphql"
+
+export const CollectorResume = new GraphQLObjectType<any, ResolverContext>({
+  name: "CollectorResume",
+  fields: () => ({
+    collectorProfile: {
+      type: CollectorProfileType,
+      resolve: ({ collector_profile }) => collector_profile,
+    },
+    hasPartnerFollow: {
+      type: GraphQLBoolean,
```

Comment:
Should be non-nullable

Reply:
We try to use `is` prefixing naming for booleans, if possible/makes sense.

Reply:
how about `isCollectorFollowingPartner`?